npm/angular/src/mount.ts

```diff
@@ -72,6 +72,15 @@ export interface MountConfig<T> extends TestModuleMetadata {
   componentProperties?: Partial<{ [P in keyof T]: T[P] }>
 }
 
+let activeFixture: ComponentFixture<any> | null = null
+
+function cleanup () {
+  activeFixture = null
+  // Not public, we need to call this to remove the last component from the DOM
+  getTestBed()['tearDownTestingModule']()
```

Comment:
Do you think we need to be a bit defensive here? Eg

```js
const teardown = getTestBed()['tearDownTestingModule']
if (!teardown) {
  throw Error('Could not teardown component. The version of Angular you are using may not be officially supported').
  // OR just return early?
  return
}

teardown()
```

Reply:
Surprised TS isn't erroring here with "might be undefined" - maybe we should make the TS setting more strict.

Reply:
I like the idea of throwing an error here. Since this is something that might break in a minor, it's best to fail early and loud. The logic in `tearDownTestingModule` is important to guarantee clean state for the next tests so I'm ok with erroring in this case.

Reply:
Addressed in [ce2c587](https://github.com/cypress-io/cypress/pull/24470/commits/ce2c5876323a89d9600bfc6bb21e539d7d9bded3), also made the types more strict for `npm/angular`